executor/src/witgen/affine_expression.rs

```diff
@@ -384,8 +413,20 @@ mod test {
         assert_eq!(b.clone() + a.clone(), a + b,);
     }
 
+    #[test]
+    pub fn test_affine_add_with_ref_key() {
+        let names = ["abc", "def", "ghi"];
+        let a = AffineExpression::<&str>::from_variable_id(names[0])
+            + FieldElement::from(2) * AffineExpression::<&str>::from_variable_id(names[1])
+            + 3.into();
+        let b = AffineExpression::<&str>::from_variable_id(names[0]) * 11.into() + 13.into();
+        let result = a.clone() + b.clone();
+        assert_eq!(&result.to_string(), "12 * abc + 2 * def + 16");
```

Comment:
where are 12 and 16 coming from?

Reply:
```
a = v0 + 2*v1 + 3
b = 11 * v0 + 13
=> a + b = 12 * v0 + 2 * v1 + 16`

Reply:
ah the `+` actually performs computation